cswinrt/strings/additions/Windows.Storage.Streams/WindowsRuntimeBufferExtensions.cs

```diff
@@ -426,7 +420,7 @@ public static Stream AsStream(this IBuffer source)
         public static byte GetByte(this IBuffer source, uint byteOffset)
         {
             if (source == null) throw new ArgumentNullException(nameof(source));
-            if (source.Capacity <= byteOffset) throw new ArgumentException(global::Windows.Storage.Streams.SR.Argument_BufferIndexExceedsCapacity, nameof(byteOffset));
+            if (source.Capacity < byteOffset) throw new ArgumentException(global::Windows.Storage.Streams.SR.Argument_BufferIndexExceedsCapacity, nameof(byteOffset));
```

Comment:
This check was actually correct, you are reading a byte, that is count=1, so from the discussion means `<=` is the correct check (alternatively `source.Capacity < byteOffset + 1`)

Reply:
Yes I agree